README.md

```diff
@@ -5,3 +5,6 @@ Weak passwords are no good. Create a Python command line application that genera
 - Passwords should be secure, must have alphanumeric characters and non-alphanumeric characters (!?#$% and so on), and should be of a certain length (up to you)
 - Passwords should also be memorable. Relevant XKCD. Maybe get your passwords from a list of generic words ( or multiple lists even)
 - Plus points if you could also design a function that checks a given string’s password strength (if the user already has a password in mind)
+- Skills Required
+    - Input/Output
+    - String Manipulation
```

Comment:
Can you add an additional skill? think we also need error handling?

Reply:
done lol